src/pages/be-square/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
```

Comment:
can we change the folder to besquare 🙏

Reply:
also link for the social icons at the footer 🙏

Reply:
Sure ✌🏻